tracer/src/Datadog.Trace/SpanTypes.cs

```diff
@@ -81,5 +81,10 @@ public static class SpanTypes
         /// The span type for db integrations (including couchbase)
         /// </summary>
         public const string Db = "db";
+
+        /// <summary>
+        /// The span type for GRPC integrations
+        /// </summary>
+        internal const string Grpc = "grpc";
```

Comment:
Are we using the same span type for client and server? (I'm aware this is different than the span name I already commented on several times.)

Reply:
Yes, this is what the [RFC](https://github.com/DataDog/architecture/blob/master/rfcs/apm/integrations/integrations/grpc/rfc.md#normalized-behaviour) requires

Reply:
This one is less problematic than `SpanKind`, but it's inconsistent with `http`/`web` for example. Again, maybe the upcoming unification of span tags will resolve this. Thanks again.